### ISSUE TITLE
Verify gravitational calculations

### ISSUE THREAD

**Issue by jacob-swanson** [jacob-swanson/NewtonGravitySim#1]

**Comment by dankirkham** [comment]
While its not really an original idea, I actually came up for an idea for a simple 2-D space sim a while ago. It would've just been a planet and a moon. I never got past thinking of ideas for the physics model, but I did find a pretty simple rocket model [here](http://en.wikipedia.org/wiki/Variable-mass_system).

TL;DR Please make a rocket.

**Comment by dankirkham** [comment]
So if you ever do want to do it, I'm pretty sure the rocket equation all breaks down to:

F_ext = F_gravity + F_drag

v_rel = g*Isp
Isp is in [seconds](http://en.wikipedia.org/wiki/Effective_exhaust_velocity#Specific_impulse_as_a_speed_.28effective_exhaust_velocity.29)
The g is constant here and does not vary with 1/r^2 because that is how Isp is defined.
"If the "amount" of propellant is given in terms of mass (such as in kilograms), then specific impulse has units of velocity. If it is given in terms of weight (such as in kiloponds or Newtons), then specific impulse has units of time. The conversion constant between the two versions of specific impulse is g." [source](http://en.wikipedia.org/wiki/Specific_impulse)
I would just make Isp a constant, but you could make it vary with r and simulate an atmosphere.
KSP also defines their [Isp in seconds](http://wiki.kerbalspaceprogram.com/wiki/Parts#Liquid_Fuel_Engines) BTW.

dm/dt = fuel mass burn rate

and thus we have the ~~momentum of~~ force on the rocket ship! Yay!

**Comment by jacob-swanson** [comment]
Verified the earth orbiting around the sun(moveable = false)
![Sun/Earth Orbit](https://f.cloud.github.com/assets/721505/1092417/970036f0-167d-11e3-8bc2-b2b4eec78294.png)
